sirepo/package_data/static/js/sirepo-utils.js

```diff
@@ -11,6 +11,32 @@ class SirepoUtils {
         return s.charAt(0).toUpperCase() + s.slice(1);
     }
 
+    static orderOfMagnitude(val, binary=false) {
+        const MAGS = ['', 'k', 'M', 'G', 'T', 'P', 'E'];
+        let i = 0;
+        let v = 0;
+        const base = binary ? 1024 : 1000;
+        for (i = 0; i < MAGS.length; ++i) {
+            v = val / Math.pow(base, i);
+            if (v < base) {
+                break;
+            }
+        }
+        return {
+            order: i,
+            suffix: MAGS[i] + (binary ? 'i' : ''),
```

Comment:
This breaks if it is larger than "E". It's maybe fine, but the error message will be confusing.

Reply:
[885738e](https://github.com/radiasoft/sirepo/pull/4736/commits/885738ee5b12c100e3da7b191ff4bbf9ab36ae0c)